src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -256,7 +257,18 @@ private Task ProcessStreamItem(HubConnectionContext connection, StreamItemMessag
             else
             {
                 bool isStreamCall = descriptor.StreamingParameters != null;
-                return Invoke(descriptor, connection, hubMethodInvocationMessage, isStreamResponse, isStreamCall);
+                if (!isStreamCall && !isStreamResponse)
```

Comment:
Do we want to make a fast path for the default when you have 1 parallel invoke?

Reply:
Yes